### PR TITLE
Elsevier parser: added address-line value for affiliation

### PR DESCRIPTION
**Pull request by ErnestaP**
* The value of affiliation was not parsed, since it was missing address-line path.
* The affiliation was parsed as empty value, that's why the default value 'HUMAN CHECK' was added.
* ref: https://github.com/cern-sis/issues-scoap3/issues/188

### CODE REVIEW

**Review by drjova — APPROVED** [review]

**Comment by drjova** [comment]
@ErnestaP there is something wrong with gh

**Comment by ErnestaP** [comment]
Github actions fixed. Tests are running in the docker container, because of stopped python2 support on git

File: requirements.txt [hunk 1]
```diff
@@ -54,15 +55,15 @@ fs==0.5.4
...
+functools32===3.2.3.post2;python_version=='2.7'
 furl==0.5.6
 future==0.18.2
 html5lib==1.1
 hyperlink==19.0.0
 idna==2.7
 IDUtils==1.1.7
 imagesize==1.2.0
-importlib-metadata==0.18
+importlib-metadata==1.0.0
```
Comment (drjova) [inline requirements.txt:66]: do we need these changes?

File: .github/workflows/pytest.yaml [hunk 2]
```diff
@@ -8,39 +8,44 @@ on:
...
         uses: codecov/codecov-action@v2
-      
-      - name: Build and push
-        id: build
-        uses: cern-sis/gh-workflows/.github/actions/docker-build@v5.5.0
-        with:
-          image: scoap3/hepcrawl
-          username: scoap3
-          password: ${{ secrets.DOCKERHUB_PASSWORD }}
-          cache: false
```
Comment (drjova) [inline .github/workflows/pytest.yaml:43]: why did you remove this? we will not be able to deploy :)

File: .github/workflows/pytest.yaml [hunk 3]
```diff
@@ -8,39 +8,44 @@ on:
...
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install coverage
+        run: pip install coverage==4.5.4
+
+      - name: Install pytest
+        run: pip install pytest==4.6.4
```
Comment (drjova) [inline .github/workflows/pytest.yaml:25]: we don't need them anymore

File: requirements.txt (hunk 1 above)
Comment (ErnestaP) [reply requirements.txt]: mm... Maybe not, because it was raising an error when we tried to run it without docker container

File: .github/workflows/pytest.yaml [hunk 4]
```diff
@@ -8,32 +8,41 @@ on:
...
+          tests
+          --cov=./
+          --cov-report=xml
+
       - name: Upload Coverage to Codecov
         uses: codecov/codecov-action@v2
-      
+
       - name: Build and push
+        if: ${{ github.event.pull_request.merged }}
```
Comment (drjova) [inline .github/workflows/pytest.yaml:45]: you can use the same as https://github.com/SCOAP3/hepcrawl/pull/109/files#diff-038d392b404f94b2ee8e3f10b6ae76784160379d9e90638f78ee65df63353221R55 :)

File: .github/workflows/pytest.yaml [hunk 5]
```diff
@@ -8,32 +8,41 @@ on:
       - master
   release:
     types: [published]
-    
+
 jobs:
-  tests:
+  test:
```
Comment (drjova) [inline .github/workflows/pytest.yaml:13]: we don't have to rename it :) Otherwise we have to change the branch protection checks

File: .github/workflows/pytest.yaml [hunk 6]
```diff
@@ -8,32 +8,41 @@ on:
...
-      - name: Install dependencies
-        run: |
-          python --version
-          python -m pip install --upgrade pip setuptools wheel
-          pip install --no-cache-dir typing==3.7.4.1
-          pip install --ignore-installed -r requirements.txt -e .[tests,docs]
-      
+          context: .
+          dockerfile: Dockerfile
+          tags: tests_image
```
[the remark this answers is not in the document]
Comment (ErnestaP) [reply .github/workflows/pytest.yaml:26]: but I am running the test in this image: https://github.com/SCOAP3/hepcrawl/blob/a490673404715539676fdf9276b96f8808d59f06/.github/workflows/pytest.yaml#L30-L39

[the remark this answers is not in the document]
Comment (drjova) [reply .github/workflows/pytest.yaml:26]: Now I see, thanks!

**Comment by drjova** [comment]
@ErnestaP could you please squash?
